index.html

```diff
@@ -118,6 +118,143 @@ <h2>
           </li>
         </ul>
       </section>
+      <section class='informative'>
+        <h2>
+          Examples
+        </h2>
+        <section>
+          <h2>
+            Locking to a specific orientation and unlocking
+          </h2>
+          <p>
+            In this example, clicking the "Lock" button makes a request to go
+            into fullscreen and then lock to the opposite orientation. If the
+            screen is not in default orientation, pressing the "Unlock" button
+            unlocks to its default orientation.
+          </p>
+          <p>
+            The developer console logs the change in orientation type and
+            angle.
+          </p>
+          <pre class='example html'>
+  &lt;script&gt;
+  function fullScreenCheck() {
+    if (document.fullscreenElement) return;
+    return document.documentElement.requestFullscreen();
+  }
+  
+  const lockButton = document.getElementById("lockBtn");
+
+  function btnOrientation() {
+    const btnOrientation = oppOrientation();
+    lockButton.textContent = `Lock to ${btnOrientation}`;
+  }
+  
+  function oppOrientation() {
+    const { type } = screen.orientation;
+    if (type.startsWith("portrait")) {
+      return "landscape";
+    }
+    return "portrait";
+  }
+
+  async function rotate() {
+    const rotateBtn = document.getElementById("rotateBtn");
+    try {
+      await fullScreenCheck();
+    } catch (err) {
+      console.error(err);
+    }
+    const newOrientation = oppOrientation();
+    await screen.orientation.lock(newOrientation);
+    btnOrientation();
+  }
+  
+  function show() {
+    const { type, angle } = screen.orientation;
+    console.log(`Orientation type is ${type} & angle is ${angle}.`);
+  }
+  
+  screen.orientation.addEventListener("change", show);
+  screen.orientation.addEventListener("change", btnOrientation);
+  window.addEventListener("load", show);
+  &lt;/script&gt;
+  
+  &lt;button onclick="rotate() id="lockBtn"&gt;
```

Comment:
If we pass "this", then we don't need to look up the button. 
```suggestion
  &lt;button onclick="rotate(this)"&gt;
```